ui/src/shared/constants/index.ts

```diff
@@ -63,6 +63,14 @@ export const CLOUD_BILLING_PATH = '/billing'
 export const CLOUD_USAGE_PATH = '/usage'
 export const CLOUD_LOGOUT_PATH = '/logout'
 
+// defaults are being added to prevent e2e failures, need to find a better solution
+export const AUTH0_DOMAIN = process.env.AUTH0_DOMAIN || 'test_domain'
```

Comment:
as the comment above mentioned, added random defaults to allow the tests to pass since this is not currently being injected as part of the testing process (and is only relevant in k8s deployment)